### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by EthannSchneider**
added the readme for other developer to collaborate faster

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+
+## Getting Started
+### Prerequisites
+
+* Docker 4.33.0
+* PHP 8.3.11
+* Xdebug v3.3.2
+* postgres 16.4
+* pgadmin 8.11.0
+
```
Comment (Guillaume1868) [inline readme.md:15]: add composer

File: readme.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+This is a project for the MAW11 course. We're supposed to rewrite [this website](https://maw-looper.mycpnv.ch)
+
+## Getting Started
+### Prerequisites
+
+* Docker 4.33.0
+* PHP 8.3.11
+* Xdebug v3.3.2
+* postgres 16.4
```
Comment (Guillaume1868) [inline readme.md]: add gitflow

**Review by Guillaume1868 — DISMISSED** [review]

File: readme.md (hunk 1 above)
Comment (Joumoune) [inline readme.md:15]: add composer version and git flow version

**Review by Joumoune — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (EthannSchneider) [reply readme.md:15]: why git flow ?

**Review by Guillaume1868 — APPROVED** [review]
LGTM

**Review by Joumoune — APPROVED** [review]

**Pull request merged**